catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/security/app/UserApplication.java

```diff
@@ -106,6 +117,26 @@ public void init() {
         },
         util::getJson);
 
+    put(
```

Comment:
Should this use the `delete` method instead?

Reply:
That makes sense